### DISCUSSION TITLE
vcpkg packages include path

### DISCUSSION THREAD

**Discussion by hros** [xmake-io/xmake#5299] [General]
Testing the `vcpkg` integration I created a simple program that uses the `fmt` package which is installed in `vcpkg`
The `xmake.lua` file is as follows:
```lua
add_rules("mode.debug", "mode.release")
add_requires("vcpkg::fmt")

target("xdemo")
set_kind("binary")
add_files("src/*.cpp")
add_packages("fmt")
```

I added a couple of lines to the default "hello world" app:
```c++
#include <iostream>
#include <fmt/core.h>

int main(int argc, char** argv) {
    std::cout << "hello world!" << std::endl;
    fmt::print("greetings from fmt\n");
    return 0;
}
```

building the project gives the error:
> error: src/main.cpp:2:10: fatal error: fmt/core.h: No such file or directory

The `fmt/core.h` exists in `$VCPKG_ROOT/installed/x64-linux/include/`
How can get `xmake` to add the `vcpkg` include path to the project?

**Comment by star-hengxing** [comment]
add_packages("vcpkg::fmt")

https://xmake.io/#/package/remote_package?id=add-vcpkg-dependency-package

**Reply by waruqi** [reply]
`add_packages("vcpkg::fmt")` or `add_requires("vcpkg::fmt", {alias = "fmt"})`
